pages/linux/whereis.md

```diff
@@ -0,0 +1,19 @@
+# whereis
+
+> Locate the binary, source, and manual page files for a command.
+
+- Locate binary, source and man pages for ssh:
+
+`whereis {{ssh}}`
+
+- Locate binary and man pages for ls:
+
+`whereis -bm {{ls}}`
+
+- Locate source of gcc and man pages for git:
```

Comment:
Somehow `-s` doesn't seem to work on my system. It works without it.
```
20:49:51-agniva-~/play/tldr-bot$whereis -s gcc -m git
gcc:
git: /usr/share/man/man1/git.1.gz
20:50:04-agniva-~/play/tldr-bot$whereis -s gcc 
gcc:
20:50:09-agniva-~/play/tldr-bot$whereis gcc 
gcc: /usr/bin/gcc /usr/lib/gcc /home/linuxbrew/.linuxbrew/bin/gcc /usr/share/man/man1/gcc.1.gz
```

Any idea why ?

Reply:
The reason is very simple. You don't have `source of gcc` on your machine.

Reply:
Ah I see. I thought that `/usr/lib/gcc` was the source. My bad, should have checked.